tests/helpers/test_collection.py

```diff
@@ -118,7 +118,7 @@ def test_id_manager() -> None:
 async def test_observable_collection() -> None:
     """Test observerable collection."""
     coll = collection.ObservableCollection(None)
-    assert coll.async_items() == []
+    assert not coll.async_items()
```

Comment:
These... however, are not the same.

This would match both:

```py
assert coll.async_items() == []
assert coll.async_items() is None
```

Meaning that now we are no longer validating it returns a list.

Reply:
I agree, and I used pylint ignore for `tests/test_core.py`.
I missed it here - so I've adjusted it.

If there are others I can amend too - but it is something that bubbles up regularly in reviews so I don't want to blanket ignore.